src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/caseevents/ProceedOfflineCallbackHandler.java

```diff
@@ -19,9 +18,8 @@
 import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
 import static uk.gov.hmcts.reform.civil.callback.CaseEvent.PROCEEDS_IN_HERITAGE_SYSTEM;
 
-@Service
 @RequiredArgsConstructor
-public class ProceedOfflineCallbackHandler extends CallbackHandler {
+public abstract class ProceedOfflineCallbackHandler extends CallbackHandler {
```

Comment:
I think that the only change we need to do in this class is to add `camundaActivityId`. It doesn't really matter what the activity id is until we don't have the same one twice in a row within single business process.

Besides, I don't think that extending this class is going to work as we want it to. Callback handler is selected based on list of events, so if we have the same in more than one handler, then it's going to pick the one that was registered last? That's what I'm assuming by looking at `CallbackHandlerFactory` constructor.